ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/WorkflowConfig.scala

```diff
@@ -39,13 +39,13 @@ object WorkflowConfig {
   final case class FooSubmissionConfig(
       numberOfInstances: Int,
       numberOfObservers: Int,
-      numberOfDivulgees: Int,
-      numberOfExtraSubmitters: Int,
+      numberOfDivulgees: Int = 0,
```

Comment:
either provide default to all arguments, or move the ones with defaults to the top

Reply:
yes I meant bottom